packetfence.rsyslog

```diff
@@ -0,0 +1,103 @@
+# RADIUS
+if $syslogtag contains "auth" and $syslogfacility-text == "local1" then -/usr/local/pf/logs/radius.log
```

Comment:
Not really.
I can't tag the syslog messages from FreeRADIUS the way I can do it from apache.
It's a kludge, but it works.